### ISSUE TITLE
$VOLTPATH/rc/default/[g]vimrc is not being generated

### ISSUE THREAD

**Issue by hupfdule** [vim-volt/volt#272]
When installing a plugin with volt, I get the following output:

```
$ volt get tpope/vim-fugitive
[INFO] Full building /home/hupfdule/.vim/pack/volt/opt directory ...
[INFO] Installing vimrc and gvimrc ...
+ github.com/tpope/vim-fugitive > installed
```

It tells me something about a vimrc and gvimrc being installed, but that is actually not the case.
My original `~/.vimrc` is still unmodifed and the directory `$VOLTPATH/rc` doesn't even exist.

I actually don't even know what should happen. The README isn't clear about that. Should volt at any time touch my `~/.vimrc`? I expect not. Should it modify an existing `$VOLTPATH/rc/defauilt/vimrc`? I expect so. But am I responsible for symlinking my real `~/.vimrc` to `$VOLTPATH/rc/default/vimrc`? I don't know.  And if volt is creating/modifying that vimrc, what does in enter there?

It should be stated more clearly in the README what the user should do here. I expected that `$VOLTPATH/rc/default/vimrc` would be created by volt and the installed packages magically get activated.

I am using the release version 0.3.5 of volt.

**Comment by mrymtsk** [comment]
As stated [here](https://github.com/vim-volt/volt/blob/master/README.md#syncing-vimpackvolt-directory-with-voltpath), if `$VOLTPATH/rc/<profile>/{vimrc.vim,gvimrc.vim}` exist, they will be copied as `$HOME/.vim/{vimrc,gvimrc}` upon `volt build`. `$HOME/.vim/{vimrc,gvimrc}` are used as runcom files if `$HOME/{.vimrc,.gvimrc}` are not found.

Therefore, if you wish to manage your `.vimrc` and/or `.gvimrc` inside the `$VOLTPATH`, you can move them to `$VOLTPATH/rc/<profile>/{vimrc.vim,gvimrc.vim}`. If not, you don't need to do anything; `volt build` never touches `$HOME/{.vimrc,.gvimrc}` and `$VOLTPATH/rc/<profile>/{vimrc.vim,gvimrc.vim}` are not necessary in that case.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by hupfdule** [comment]
Well, it is correct what you say and it is working. But still I think the README should be refined to describe this topic a bit better. It is not easy to identify the necessary bits out of the README to create a new volt config from scratch.

**Comment by mrymtsk** [comment]
I agree that README can be more precise and friendly. I will try and make a PR when I have time :)

**Comment by tyru** [comment]
fixed in #292
